### PR TITLE
Add WordApiDesktop 1.1 requirement set and build versions

### PR DESCRIPTION
**Pull request by jipyua**

### CODE REVIEW

File: mapping/requirements_officejs.json [hunk 1]
```diff
@@ -2874,6 +2874,17 @@
...
+						{
+							"name": "WordApiDesktop",
+							"apiVersion": "1.1",
+							"supportedProductVersions": [{
+								"from": {
+									"build": "16.85.0.0",
+									"version": null
+								}
+							}],
+							"availability": "GA"
```
Comment (jipyua) [inline mapping/requirements_officejs.json:2886]: just a placeholder, will wait for July fork build number later. Don't merge so far.

File: mapping/requirements_officejs.json [hunk 2]
```diff
@@ -5088,6 +5110,17 @@
 							}],
 							"availability": "GA"
 						},
+						{
+							"name": "WordApiDesktop",
+							"apiVersion": "1.1",
+							"supportedProductVersions": [{
+								"from": {
+									"build": "16.0.17628.20114",
```
Comment (ElizabethSamuel-MSFT) [inline mapping/requirements_officejs.json:5118]: Seems like this should be 17928 based on https://learn.microsoft.com/en-us/officeupdates/update-history-microsoft365-apps-by-date.

```suggestion
									"build": "16.0.17928.20114",
```

**Comment by ElizabethSamuel-MSFT** [comment]
@jipyua The main purpose of this JSON file is for Store validation of the add-in manifest. If devs can't include WordApiDesktop 1.1 in the manifest, then we shouldn't make this change.

**Comment by jipyua** [comment]
> @jipyua The main purpose of this JSON file is for Store validation of the add-in manifest. If devs can't include WordApiDesktop 1.1 in the manifest, then we shouldn't make this change.

Thanks Elizabeth, I also didn't see hidden document requirement set in the file, so I will not check in this.

**Pull request closed without merging**